src/types.ts

```diff
@@ -1115,5 +1115,56 @@ export interface SearchReponse {
 }
 
 export interface TransactionResult {
-  txHash: string
-}
\ No newline at end of file
+  txHash: string;
+}
+
+export interface ChatConfigureParamsBase {
+  objectId: string;
+}
+
+export interface OpenAIChatConfigureParams extends ChatConfigureParamsBase {
```

Comment:
(suggestion) can this be just merged into ChatConfigureParamsBase? If we support more providers and apis, maybe more values will be added like "provider: 'openai' | 'google'; api: 'assistants' | 'assistants_v2';"?

Reply:
Yes! If we are to cover Google case, I thought of expanding it as follows:
This interface is suitable when the types of APIs provided vary depending on the provider.
But if we have to unify under the same name, it would be better to merge.

```
export interface GoogleChatConfigureParams extends ChatConfigureParamsBase {
    provider: 'google';
    api: 'gemeni' | 'others';
}

export type ChatConfigureParams = OpenAIChatConfigureParams | GoogleChatConfigureParams;
```

Reply:
Is that meant to suggest that we would better to start with a simper now and think about dividing it later if needed?

Reply:
I think the GoogleChatConfigureParams idea makes sense. Thanks!

Reply:
Thank you for comment on this code!